data.sql

```diff
@@ -1,5 +1,15 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+(1, 'Agumon', '2020-02-03', 0, 'TRUE', 10.23),
+(2, 'Gabumon', '2018-11-15', 2, 'TRUE', 8),
+(3, 'Pikachu', '2021-01-07', 1, 'FALSE', 15.04),
+(4, 'Devimon', '2017-05-12', 5, 'TRUE', 11),
+(5, 'Charmander', '2020-02-08', 0, 'FALSE', 11),
```

Comment:
- [ ] Great job so far, but you need to pay more attention to the project requirements. The first data must have a negative weight_kg value and not a positive value. Please fix this 👍🏻Your code might look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');
```